content/ADR-144.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: adr
+adr: 144
```

Comment:
```suggestion
adr: 150
```
And also rename the file

Reply:
The issue was created to park the ID 144